caduceus.go

```diff
@@ -74,7 +78,7 @@ func caduceus(arguments []string, run bool) error {
 			goschtalt.UnmarshalFunc[sallust.Config]("logging"),
 			goschtalt.UnmarshalFunc[candlelight.Config]("tracing"),
 			goschtalt.UnmarshalFunc[touchstone.Config]("prometheus"),
-			goschtalt.UnmarshalFunc[SinkConfig]("sender"),
+			goschtalt.UnmarshalFunc[sink.SinkConfig]("sender"),
```

Comment:
a general note to help with readability , might be worth bundling fx options in respective package, e.g `sink.Provide()`.

We can always do this later if you want to punt it